web/styles/portico/markdown.css

```diff
@@ -392,6 +396,19 @@
         }
     }
 
+    /* Style copy-to-clipboard button inside code blocks */
+    .copy-codeblock {
+        /* Invisible default button */
+        background: none;
+        border: none;
+        padding: 0;
+        display: inline-block;
```

Comment:
It should be safe to strike the `inline-block` line here. Buttons display as `inline-block` by default, but with the absolute positioning here, there should be no effect from this property.